activefires_pp/post_processing.py

```diff
@@ -616,10 +616,9 @@ def update_fire_detection_id(self):
         """Update the fire detection ID registry."""
         now = datetime.utcnow()
         tdelta = now - self._fire_detection_id['date']
-        if tdelta.total_seconds() > 24*3600:
-            self._initialize_fire_detection_id()
-        elif tdelta.total_seconds() > 0 and self._fire_detection_id['date'].day != now.day:
-            self._initialize_fire_detection_id()
+        if (tdelta.total_seconds() > 24*3600 or
+                (tdelta.total_seconds() > 0 and self._fire_detection_id['date'].day != now.day)):
```

Comment:
I'm not sure I get it right, but is this not equivalent to:
```suggestion
        if self._fire_detection_id['date'].date() < now.date():
```
?

Reply:
You are right! Thanks!